openshift-ci/Dockerfile.tools

```diff
@@ -6,6 +6,8 @@ ENV GOVERSION=1.16.3
 ENV GOPATH=/go
 ENV GOBIN=${GOPATH}/bin
 ENV PATH=${PATH}:${GOROOT}/bin:${GOBIN}
+ENV GO111MODULE=on
+ENV GINKGO_VERSION v1.12.1
```

Comment:
why is the convention here is different from the rest ?
`ENV GINKGO_VERSION v1.12.1`
and not `ENV GINKGO_VERSION=v1.12.1 ` ? 
(might not have a negative affect but still)

Reply:
I have just used the same line as in the previous branches, so no strong reason so far.
Will change to keep it homogeneous.

Reply:
Well, it is a bit annoying indeed but it's cosmetic more than functional: https://docs.docker.com/engine/reference/builder/#env (see: "alternatyve syntax")
next time let's be consistent with the surrounding code, but it's not a big deal (and PAO is already too messy already; OTOH that's not a reason to increase the mess...)